src/main/java/fiftyfive/administration/usermanagement/controller/UserManagementController.java

```diff
@@ -7,42 +7,48 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
-
-import java.util.ArrayList;
 import java.util.List;
 
 @RestController
 @RequestMapping("/v1/users")
 public class UserManagementController {
-    List<User> userList = new ArrayList<>();
 
     @Autowired
     UserService userService;
 
     @PostMapping()
     public ResponseEntity<UserResponseData> createUser(@RequestBody @Valid CreateUserRequest createUserRequest) {
-        return ResponseEntity.status(HttpStatus.CREATED).body(userService.createUser(createUserRequest, userList));
+        return ResponseEntity.status(HttpStatus.CREATED).body(userService.createUser(createUserRequest));
     }
 
     @PutMapping("/{username}")
     public ResponseEntity<UserResponseData> updateUser(@RequestBody UpdateUserRequestData userRequestData, @PathVariable String username) {
-        return ResponseEntity.status(HttpStatus.OK).body(userService.updateUser(userRequestData, userList, username));
+        return ResponseEntity.status(HttpStatus.OK).body(userService.updateUser(userRequestData, username));
 
     }
 
     @GetMapping("/{username}")
     public ResponseEntity<UserResponseData> getUser(@PathVariable String username) {
-        return ResponseEntity.status(HttpStatus.OK).body(userService.getUser(username, userList));
+        return ResponseEntity.status(HttpStatus.OK).body(userService.getUser(username));
     }
 
     @DeleteMapping("/{username}")
     public ResponseEntity<UserResponseData> deleteUser(@PathVariable String username) {
-        return ResponseEntity.status(HttpStatus.NO_CONTENT).body(userService.deleteUser(username, userList));
+        return ResponseEntity.status(HttpStatus.NO_CONTENT).body(userService.deleteUser(username));
     }
 
     @GetMapping()
     public ResponseEntity<List<UserResponseData>> getAllUser() {
-        List<UserResponseData> user = userService.getAllUsers(userList);
+        List<UserResponseData> user = userService.getAllUsers();
+        if (user.isEmpty()) {
+            return ResponseEntity.status(HttpStatus.NO_CONTENT).body(user);
```

Comment:
```suggestion
            return ResponseEntity.status(HttpStatus.NO_CONTENT);
```

If the status is no content we don't set any body